test/index.test.ts

```diff
@@ -108,5 +108,32 @@ describe('index', () => {
     it('throws error when no platform specified', async () => {
       await expect(generateNotes({}, generateNotesContext)).rejects.toThrowErrorMatchingSnapshot('noPlatform')
     })
+    it('adds heading when specified', async () => {
+      const heading = 'Example Heading'
+      const { iOS, android } = pluginConfig.platforms
+
+      const expectedHeading = `### ${heading}`
+      const expectedAndroid = `${android.displayName} Version Range: **\`>= 1.2.3 and < 4.5.6\`**`
+      const expectedIOS = `${iOS.displayName} Version Range: **\`>= 1.2.3 and < 4.5.6\`**`
+
+      const subTestCases = [
```

Comment:
We could generate separate jest test-case like this:
```
const subTestCases = [...]
subTestCases.forEach(testCase => {
  it(`adds heading when specified for platforms: ${testCase.platforms}`) {
   ...
   }
})
```

That way, when one case fails, it's easier to figure out which one it is 🙂

Reply:
Thanks a lot. I've updated it. Now, it's much more maintainable, and we can understand exactly in which case it fails